  <!--==========================
    Page Banner Section
  ============================-->
  <section id="innerBanner"> 
    <div class="inner-content">
      <h2><span>How to Get Gene Sequence</span></h2>
      <div> 
      </div>
    </div> 
  </section><!-- #Page Banner -->

  <main id="main">

    <!--==========================
      About Section
    ============================-->
    <section id="about" class="wow fadeInUp">
      <div class="container"> 
        <div class="row">
          
          <div class="col-lg-6 content">
            <h5>Sanger's Method<h3>(Basic Gene Sequence Technique)</h3></h5>
			<p class="text-justify">Sanger sequencing, also known as the “chain termination method”, is a method for determining the nucleotide sequence of DNA. The method was developed by two time Nobel Laureate Frederick Sanger and his colleagues in 1977, hence the name the Sanger Sequence.Sanger sequencing is a method of DNA sequencing based on the selective incorporation of chain-terminating dideoxynucleotides by DNA polymerase during in vitro DNA replication. Developed by Frederick Sanger and colleagues in 1977, it was the most widely used sequencing method for approximately 40 years.Sanger sequencing works on the principle that when given enough time and enough starting material, at least one DNA sequence of every possible length will be produced with a tagged nucleotide at the end. The tagged nucleotide will always terminate the sequence because dideoxynucleotides are missing the 3'-OH group that is necessary to continue the chain</p>
            
          </div>
		  <div class="col-lg-6 about-img">
            <img src="/assets/images/sanger.png" alt="">
          </div>

        </div>

      </div>
    </section><!-- #about -->
	    <section id="about" class="wow fadeInUp">
      <div class="container"> 
        <div class="row">
          
          <div class="col-lg-6 content">
            <h5>Maxam-Gilbert Method<h3>(Advance Gene Sequence Technique)</h3></h5>
			<p class="text-justify">Maxam–Gilbert sequencing is a method of DNA sequencing developed by Allan Maxam and Walter Gilbert in 1977–1980. This method is based on nucleobase-specific partial chemical modification of DNA and subsequent cleavage of the DNA backbone at sites adjacent to the modified nucleotides.Maxam–Gilbert sequencing requires radioactive labeling at one 5′ end of the DNA fragment to be sequenced (typically by a kinase reaction using gamma-32P ATP) and purification of the DNA. Chemical treatment generates breaks at a small proportion of one or two of the four nucleotide bases in each of four reactions (G, A+G, C, C+T). For example, the purines (A+G) are depurinated using formic acid, the guanines (and to some extent the adenines) are methylated by dimethyl sulfate, and the pyrimidines (C+T) are hydrolysed using hydrazine. The addition of salt (sodium chloride) to the hydrazine reaction inhibits the reaction of thymine for the C-only reaction. The modified DNAs may then be cleaved by hot piperidine; (CH2)5NH at the position of the modified base. The concentration of the modifying chemicals is controlled to introduce on average one modification per DNA molecule. Thus a series of labeled fragments is generated, from the radiolabeled end to the first "cut" site in each molecule.

The fragments in the four reactions are electrophoresed side by side in denaturing acrylamide gels for size separation. To visualize the fragments, the gel is exposed to X-ray film for autoradiography, yielding a series of dark bands each showing the location of identical radiolabeled DNA molecules. From presence and absence of certain fragments the sequence may be inferred.</p>
            
          </div>
		  <div class="col-lg-6 about-img">
            <img src="/assets/images/Max.jpg" alt="">
          </div>

        </div>

      </div>
    </section>
	      <section id="about" class="wow fadeInUp">
      <div class="container"> 
        <div class="row">
          
          <div class="col-lg-6 content">
            <h5>Pyro-Sequencing Method<h3>(Next Generation Gene Sequence Technique)</h3></h5>
			<p class="text-justify">Pyrosequencing is a method of DNA sequencing based on the "sequencing by synthesis" principle, in which the sequencing is performed by detecting the nucleotide incorporated by a DNA polymerase. Pyrosequencing relies on light detection based on a chain reaction when pyrophosphate is released.Pyrosequencing is one of the DNA sequencing techniques and has numerous potential applications.46–48 It is based on the chain of enzymatic reactions during DNA synthesis when the inorganic pyrophosphate (PPi) is released as a result of nucleotide incorporated by polymerase. The light is generated as the result of luciferin oxidation by luciferase. The energy for this reaction is supplied via conversion of pyrophosphate to adenosine triphosphate (ATP). According to Ronaghi,49 the overall reaction from polymerization to light detection usually takes 3–4 s at room temperature; 1 pmol of DNA could yield 6 × 1011 ATP molecules, which, in turn, generate more than 6 × 109 photons at a wavelength of 560 nm. The light is detected by a photodiode, photomultiplier tube, or a charge-coupled device (CCD) camera </p>
            
          </div>
		  <div class="col-lg-6 about-img">
            <img src="/assets/images/pyro.jpg" alt="">
          </div>

        </div>

      </div>
    </section>
    <!--==========================
      Clients Section
    ============================-->
 

  <a href="#" class="back-to-top"><i class="fa fa-chevron-up"></i></a>

 

